app/screens/global_drafts/components/draft_empty_component.tsx

```diff
@@ -1,17 +1,15 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {Image} from 'expo-image';
 import React from 'react';
 import {View} from 'react-native';
 
 import FormattedText from '@components/formatted_text';
+import DraftEmpty from '@components/illustrations/draft_empty';
 import {useTheme} from '@context/theme';
 import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
 import {typography} from '@utils/typography';
 
-const draft_message_image = require('@assets/images/Draft_Message.png');
```

Comment:
Delete the asset if no longer used :)

Reply:
I haven't even finished the PR description 🤣